src/applications/caregivers/definitions/UIDefinitions/veteranUI.js

```diff
@@ -133,15 +135,14 @@ export const LastTreatmentFacilityAPIUI = {
 };
 
 export const PreferredFacilityAPIUI = {
-  'ui:title': 'VA health care services',
-  'ui:description': PreferredFacilityDescription,
-  veteranFacilityState: {
-    'ui:title': 'State',
-    'ui:required': () => true,
-  },
+  'ui:title': 'The Veteran’s VA health care facility',
+  'ui:description': PreferredFacilityAPIDescription,
   plannedClinic: {
-    'ui:title': 'VA medical center',
-    'ui:widget': VaMedicalCenter,
+    'ui:title': 'Search by city, state or postal code',
```

Comment:
Since were hiding the label text in the ui:options below, we shouldn't need this title attribute.